.env

```diff
@@ -58,10 +61,6 @@ SEARCH_ENGINE_ID="Fill your API key here"
 # OpenWeatherMap API (https://home.openweathermap.org/users/sign_up)
 OPENWEATHERMAP_API_KEY="Fill your API key here"
 
-# NVIDIA API (https://build.nvidia.com/explore/discover)
-NVIDIA_API_BASE_URL="Fill your API key here"
```

Comment:
I believe that we still need this.

Reply:
Hi Will, thanks for your commenting. I noticed that Qwen model did this so I copied their pattern. I will fix this by adding the line back.